iaso/migrations/0239_add_feature_flag_ORG_UNIT_CHANGE_REQUEST.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.22 on 2023-10-25 09:36
+
+from django.db import migrations
+
+
+def create_feature_flags(apps, schema_editor):
+    FeatureFlag = apps.get_model("iaso", "FeatureFlag")
+    FeatureFlag.objects.create(
+        code="ORG_UNIT_CHANGE_REQUEST",
+        name="Request changes to org units.",
+    )
+
+
+def destroy_feature_flags(apps, schema_editor):
+    FeatureFlag = apps.get_model("iaso", "FeatureFlag")
+    FeatureFlag.objects.filter(code="ORG_UNIT_CHANGE_REQUEST").delete()
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("iaso", "0238_rename_new_accuracy_orgunitchangerequest_new_location_accuracy_uuid"),
+    ]
+
+    operations = [migrations.RunPython(create_feature_flags, destroy_feature_flags)]
```

Comment:
Should the feature flag be used by the backend code? Or is it only for clients usage?

Reply:
It's good to be able to have the feature activated only on some projects (= some apps), so it's ok like it is now. If we want to hide the feature on web too, we'll desactivate the module for the account (the current PR of Franck)